### PR TITLE
Primecoin Definition

### PR DESCRIPTION
**Pull request by thedevworks**
Addition of Primecoin Definition and Icon. 

Thanks!

### CODE REVIEW

File: defs/bitcoin/primecoin.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "coin_name": "Primecoin",
+  "coin_shortcut": "XPM",
+  "coin_label": "Primecoin",
+  "website": "https://primecoin.io",
+  "github": "https://github.com/primecoin",
```
Comment (prusnak) [inline defs/bitcoin/primecoin.json]: change this to https://github.com/primecoin/primecoin

**Comment by prusnak** [comment]
Show me a Primecoin explorer proving this coin is still alive.

File: defs/bitcoin/primecoin.json (hunk 1 above)
Comment (thedevworks) [reply defs/bitcoin/primecoin.json]: Ok this is done

**Comment by thedevworks** [comment]
The coin is very much still alive, infact its original founder Sunny King (Proof of Stake, Primecoin, Peercoin, Vee) has returned to assist in development and is establishing a foundation for ongoing development. 

The block explorer can be found here; 

http://explorer.primecommunity.org/

File: defs/bitcoin/primecoin.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "fork_id": null,
+  "force_bip143": false,
+  "bip115": false,
+  "default_fee_b": {
+    "Normal": 10
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 60,
+  "uri_prefix": "primecoin",
+  "min_address_length": 35,
```
Comment (prusnak) [inline defs/bitcoin/primecoin.json]: How did you come up with values for min_address_length and max_address_length? I picked a random address from the explorer and it was 34 characters long. Please adjust the values to correct ones.

Comment (thedevworks) [reply defs/bitcoin/primecoin.json]: Corrected! Apologies for the oversight.. I shouldn't do PR's at 2am! ;)

**Pull request merged**